extensions/librdkafka/PublishKafka.cpp

```diff
@@ -619,7 +622,7 @@ bool PublishKafka::createNewTopic(const std::shared_ptr<core::ProcessContext> &c
   }
 
   // The topic takes ownership of the configuration, we must not free it
-  gsl::owner<rd_kafka_topic_t*> topic_reference = rd_kafka_topic_new(conn_->getConnection(), topic_name.c_str(), topic_conf_.release());
+  const auto topic_reference = gsl::owner<rd_kafka_topic_t*>(rd_kafka_topic_new(conn_->getConnection(), topic_name.c_str(), topic_conf_.release()));  // NOLINT(cppcoreguidelines-owning-memory)
```

Comment:
It was needed due to the `conn_->getConnection()` parameter as it returned gsl::owner instead of a raw pointer. I changed the return type to return raw pointer without changing ownership in 760f6ab8125f79b5454684d6519442d725690a5c